packages/mui-joy/src/Button/ButtonProps.ts

```diff
@@ -107,7 +107,7 @@ export interface ButtonTypeMap<P = {}, D extends React.ElementType = 'button'> {
        */
       variant?: OverridableStringUnion<VariantProp, ButtonPropsVariantOverrides>;
       /**
-       * If `true`, the loading indicator is shown.
+       * If `true`, the loading indicator is shown and the button becomes disabled.
```

Comment:
We should fix Material UI as well no?

Reply:
Yes, we could fix it for Loading Button component which is under MUI lab.

Reply:
Okay I'll open a PR